frontend/src/pages/OverviewStandardsPage.vue

```diff
@@ -0,0 +1,266 @@
+                    <li>Raw under <code>acquisition</code>; derived under <code>processing</code></li>
+                  </ul>
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+            <q-item>
+              <q-item-section>
+                <q-item-label overline>Naming & metadata</q-item-label>
+                <q-item-label>
+                  <ul class="q-ml-lg q-mt-sm">
+                    <li>Use object types/names from PyNWB/MatNWB</li>
+                    <li>Record units, sampling rates, and reference time consistently</li>
+                    <li>Leverage NDX extensions when needed</li>
+                  </ul>
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+            <q-item>
+              <q-item-section>
+                <q-item-label>
+                  Learn more: <a href="https://nwb.org/about-nwb/" target="_blank" rel="noopener" class="link">NWB</a>
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+          </q-list>
+        </q-card-section>
+      </q-card>
+
+      <!-- Hybrid Layout Example -->
+      <q-card flat class="q-mt-lg">
+        <q-card-section>
+          <div class="text-h2 scrollable">Hybrid Layout Example</div>
+          <q-list>
+            <q-item>
+              <q-item-section>
+                <q-item-label overline>BIDS + NWB</q-item-label>
+                <q-item-label>
+                  Store <code>.nwb</code> files within a BIDS directory structure (e.g., iEEG or behavior):
+                </q-item-label>
+                <q-item-label>
+                  <code class="q-pa-xs bg-grey-2">sub-01/ses-01/ieeg/sub-01_ses-01_task-rest_run-01_ieeg.nwb</code>
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+          </q-list>
+        </q-card-section>
+      </q-card>
+
+      <!-- Validation Tools & Services (uniform presentation) -->
+      <q-card flat class="q-mt-lg">
+        <q-card-section>
+          <div class="text-h2 scrollable">Validation Tools & Services</div>
+          <q-list>
+            <q-item>
+              <q-item-section>
+                <q-item-label overline>BIDS Validator</q-item-label>
+                <q-item-label>
+                  Check directory layout, filenames, and sidecars, then iterate until clean.
+                </q-item-label>
+                <q-item-label>
+                  <a href="https://bids-standard.github.io/bids-validator/" target="_blank" rel="noopener" class="link">Open BIDS Validator</a>
+                </q-item-label>
+              </q-item-section>
+            </q-item>
+            <q-item>
+              <q-item-section>
+                <q-item-label overline>NWB Validation</q-item-label>
+                <q-item-label>
+                  Validate with <code>pynwb validate</code> or NWBInspector for schema compliance and metadata completeness.
```

Comment:
```suggestion
                  Validate with <a href="https://nwbinspector.readthedocs.io/en/dev/index.html" target="_blank" rel="noopener" class="link">NWB Inspector</a> for schema compliance and metadata completeness.
```

To have one fewer thing to 'need to know'

FYI the NWB Inspector also runs and reports `pynwb validate`